dolphin/db/api.py

```diff
@@ -142,6 +147,11 @@ def volume_get_all(context, marker=None, limit=None, sort_keys=None,
                                sort_dirs, filters, offset)
 
 
+def volume_delete(context, storage_id):
```

Comment:
it's better to rename it to volume_delete_by_storage

Reply:
Done